TestingGrounds/TestingGrounds/HashTable.cpp

```diff
@@ -63,26 +60,80 @@ uint32_t HashTable<K, V>::calcPrime( void )
     return ret;
```

Comment:
This will always return `0` since everything in the `prime` array is `false`.

Reply:
Changed!